debian/install

```diff
@@ -0,0 +1,2 @@
+share usr
+debian/pkg.rb usr/lib/ruby/vendor_ruby/oswitch
```

Comment:
@tbooth: Does `usr/lib/ruby/vendor_ruby/oswitch` have to be hard-coded here? Is there some magic variable that I can use instead?